arex-instrumentation/redis/arex-redis-common/src/main/java/io/arex/inst/redis/common/RedisKeyUtil.java

```diff
@@ -23,6 +24,22 @@ public static <K, V> String generate(Map<K, V> map) {
         return generate(map.keySet());
     }
 
+    public static String generateOne(Object key) {
```

Comment:
change name to generate and the above two methods of this method can be changed to private.